snapshots/snapshotter.go

```diff
@@ -160,9 +160,13 @@ func (u *Usage) Add(other Usage) {
 //	layerPath, tmpDir := getLayerPath(), mkTmpDir() // just a path to layer tar file.
 //
 // We start by using a Snapshotter to Prepare a new snapshot transaction, using a
-// key and descending from the empty parent "":
+// key and descending from the empty parent "". To prevent our layer from being
+// garbage collected during unpacking, we add the `gc.root` label:
```

Comment:
We might want to call out the full label name to be explicit in case someone doesn't look at the example :)

Reply:
Yep fixed!